packages/SwingSet/src/kernel/vatManager/vat-warehouse.js

```diff
@@ -58,6 +59,8 @@ export const makeLRU = max => {
  */
 export function makeVatWarehouse(kernelKeeper, vatLoader, policyOptions) {
   const {
+    // Whether to eliminate all vat nodeterminism.
+    consensusMode = false,
```

Comment:
`warehousePolicy` was kind of meant to define the RAM-vs-time tradeoffs of the warehouse itself, rather than to be a way to control the execution of vats.. it feels like `consensusMode` wants to be part of some other options object (which the VatWarehouse may need to be aware of, to instruct VatManagers how to work). I'm not sure I have a better recommendation, though.

Reply:
Noted.

Reply:
I created a new kernel runtime option called `overrideVatManagerOptions`, which does as advertised.  `consensusMode` now lives in there, instead of in the VatWarehouse.